### DISCUSSION TITLE
Supermaster/Superslave with mySQL and Slave update always refresh all into slave with an infinite records.id increment

### DISCUSSION THREAD

**Discussion by carmas123** [PowerDNS/pdns#13512] [Q&A]
Hi, I've a Supermaster and a Superslave configurated and all work fine.
Each server use a distinct mySQL database obviously, but I see the when the slave receive a zone commit it recreate all records into table and the AUTOINC field of records table is always increased.

There is a way to update all changed records without recreate the entire table?

Regards.

**Comment by omoerbeek** [comment]
The authoritative server can do IXFR (incremental updates) in a client (secondary) role (by setting the `IXFR` meta data item on the zone), but the server side is not implemented yet. It *is* high on the wish list.

In the meantime, if the large updates are an issue for your use-case, `[ixfrdist`](https://docs.powerdns.com/authoritative/manpages/ixfrdist.1.html) might be suited. It offers the server side of IXFR by doing an AXFR form the authoritative server and then computing and serving the deltas.
